node/pkg/ethereum/watcher.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/certusone/wormhole/node/pkg/p2p"
 	gossipv1 "github.com/certusone/wormhole/node/pkg/proto/gossip/v1"
 	"github.com/ethereum/go-ethereum/rpc"
-	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promauto"	
```

Comment:
You added a trailing tab here. You can manually remove it or run `gofmt`.

Reply:
Doh! I lost the setting to make diff show trailing spaces when my mac got reimaged. Fixed!